simulator.py

```diff
@@ -239,13 +237,13 @@ def symptoms(self):
         return symptoms
 
     @property
-    def infectiousness(self):
+    def viral_load(self):
+        """ Calculates the elapsed time since infection, returning this person's current viral load"""
         if self.is_infectious:
-            days_exposed = (self.env.timestamp - self.infection_timestamp).days
-            if days_exposed > len(INFECTIOUSNESS_CURVE):
-                return 0
-            else:
-                return INFECTIOUSNESS_CURVE[days_exposed - 1]
+            time_exposed = (self.env.timestamp - self.infection_timestamp)
+            time_exposed_days = time_exposed.days + time_exposed.seconds / 86400 #(seconds in a day)
+            cur_viral_load = self.viral_load_dist.pdf(time_exposed_days)
+            return cur_viral_load
```

Comment:
yes -- @teganmaharaj and I discussed briefly this idea and I think we agreed that they should be modelled seperately.

Reply:
One other thing to note: I think we're modelling "is_infectious" incorrectly, and that will mess this code up. I can put a limit like VIRAL_LOAD_MIN=0.0001 and check `if cur_viral_load < VIRAL_LOAD_MIN: return 0` to fix it, but otherwise we will always be returning small nonzero values after a person has become infected.

that property is created here: 
```
    def is_infectious(self):
        return self.infection_timestamp is not None and self.env.timestamp - self.infection_timestamp >= datetime.timedelta(days=self.incubation_days)
```

Maybe I could change it to depend on cur_viral_load?
```
    def is_infectious(self):
        return self.infection_timestamp is not None and self.env.timestamp - self.infection_timestamp >= datetime.timedelta(days=self.incubation_days) and self.viral_load > VIRAL_LOAD_MIN
```

One worry is that this flag may be mis-used downstream somewhere to create a count of the number of people who have been infected, though clearly with a name like `is_infectious`, it should not be used that way.

Reply:
yes; according to abhinav we do (viral load is the gamma distribution, and infectiousness is related to this curve but is based on "viral shedding" and also severity of symptoms. Basically if you have a weak immune system / are older, you'll shed more (i.e. will be more contagious) over a longer period of time.